app/views/categories/index.html.erb

```diff
@@ -0,0 +1,39 @@
+<section class="categories-container">
+  <div class="categories-header">
+      <div class ="exit">
+      <%= button_to 'Sign out', destroy_user_session_path, method: :delete, class: 'delete-btn' %>
+       </div>
+      <h2>CATEGORIES</h2>
+      <div class ="search">
+        <%= image_tag "search.png", alt: "Search Icon", class: "search-img"%>
+      </div>
+  </div>
+  
+  <ul class="categories">
+ 
+  <% @categories.each do |category| %>
+    <% if user_signed_in? && category.user_id == current_user.id %>
+     <li class="category-container">
+     <div class="cat-icon">
+     <%= image_tag category.icon, alt: "category Icon", class: "cat-image"%>
+     </div>
+        <div class="cat-info-cont">
+            <h3>
+                <%= link_to category.name, category_path(category), { class: "category-name" } %>
+            </h3>
```

Comment:
- [ ] Please style the category name name according to the design guidelines. Kindly remove the `text-decoration`, and ensure that the text-color matches the given design's 👍 
## Your Design
![image](https://user-images.githubusercontent.com/46839548/236615994-273701de-7994-4d80-aadf-bd0421d8c338.png)
## Design Guideline
![image](https://user-images.githubusercontent.com/46839548/236616062-9ff7af4a-13ba-4bee-8603-f73b09cae8cc.png)